addons/mrp/models/mrp_bom.py

```diff
@@ -541,8 +532,7 @@ def _get_default_product_uom_id(self):
         'uom.uom', 'Product Unit of Measure',
         default=_get_default_product_uom_id,
         required=True,
-        help="Unit of Measure (Unit of Measure) is the unit of measurement for the inventory control", domain="[('category_id', '=', product_uom_category_id)]")
-    product_uom_category_id = fields.Many2one(related='product_id.uom_id.category_id')
+        help="Unit of Measure (Unit of Measure) is the unit of measurement for the inventory control")
```

Comment:
This help message doesn't make sense + is very poorly written. We may as well delete it while we're touching the line